### PR TITLE
[JBTM-3699] change of path in run.sh for the quickstart lra-examples

### PR DESCRIPTION
**Pull request by jmfinelli**
Our CI doesn't like relative paths. This PR is the workaround. I already [tested](https://ci-jenkins-csb-narayana.apps.ocp-c1.prod.psi.redhat.com/job/narayana_6.0-quickstarts/7/) this PR (the quickstart with MongoDB fails because there isn't any docker/podman environment in our CI)

### CODE REVIEW

**Review by marcosgopen — APPROVED** [review]
Nice spotted @jmfinelli! Thanks for the PR

File: rts/lra-examples/run.sh [hunk 1]
```diff
@@ -47,14 +47,6 @@ function killpid {
   test $? || echo "===== could not kill $1"
 }
 
-function start_coordinator {
```
Comment (mmusgrov) [inline rts/lra-examples/run.sh:50]: @jmfinelli why have you removed this function for starting the coordinator.
The purpose of a quickstart is help the user become familiar with how to use the s/w - now it's less clear how to start coordinators. The purpose of functions is to create modular readable software, can you re-instate it please.

Comment (jmfinelli) [reply rts/lra-examples/run.sh:50]: I removed this function as it was not used. The same code was already pasted in the `if` at (the old) code line 125. If you think we should keep it anyway, I'll re-instate it

Comment (mmusgrov) [reply rts/lra-examples/run.sh:50]: If it's not used then you were right to remove it, sorry.

**Pull request merged**